cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/OutletDaoCompoundIT.java

```diff
@@ -0,0 +1,333 @@
+
+    private static final Outlet BASE_AND_SUB_LOC_COMPOUND_OUTLET = buildTestOutlet(COMPOUND_OUTLET_2_LOC, PROJECT_LOC2,
+                                                                                   buildCompoundOutletRecord(CO2_INTAKE,
+                                                                                                             CO2_WEIR,
+                                                                                                             CO2_CONDUIT),
+                                                                                   buildCompoundOutletRecord(CO2_WEIR),
+                                                                                   buildCompoundOutletRecord(
+                                                                                           CO2_CONDUIT));
+
+    private static final Outlet BASE_LOC_ONLY_COMPOUND_OUTLET = buildTestOutlet(COMPOUND_OUTLET_3_LOC, PROJECT_LOC2,
+                                                                                buildCompoundOutletRecord(CO3_I1,
+                                                                                                          CO3_CONDUIT),
+                                                                                buildCompoundOutletRecord(CO3_I2,
+                                                                                                          CO3_CONDUIT),
+                                                                                buildCompoundOutletRecord(CO3_I3,
+                                                                                                          CO3_CONDUIT),
+                                                                                buildCompoundOutletRecord(CO3_CONDUIT));
+    private static final Outlet CO1_I25_OUTLET = buildTestOutlet(CO1_I25, PROJECT_LOC2);
+    private static final Outlet CO1_I53_OUTLET = buildTestOutlet(CO1_I53, PROJECT_LOC2);
+    private static final Outlet CO1_LOW_FLOW_OUTLET = buildTestOutlet(CO1_LOW_FLOW, PROJECT_LOC2);
+    private static final Outlet CO2_CONDUIT_OUTLET = buildTestOutlet(CO2_CONDUIT, PROJECT_LOC2);
+    private static final Outlet CO2_INTAKE_OUTLET = buildTestOutlet(CO2_INTAKE, PROJECT_LOC2);
+    private static final Outlet CO2_WEIR_OUTLET = buildTestOutlet(CO2_WEIR, PROJECT_LOC2);
+    private static final Outlet CO3_I1_OUTLET = buildTestOutlet(CO3_I1, PROJECT_LOC2);
+    private static final Outlet CO3_I2_OUTLET = buildTestOutlet(CO3_I2, PROJECT_LOC2);
+    private static final Outlet CO3_I3_OUTLET = buildTestOutlet(CO3_I3, PROJECT_LOC2);
+    private static final Outlet CO3_CONDUIT_OUTLET = buildTestOutlet(CO3_CONDUIT, PROJECT_LOC2);
+
+    @BeforeAll
+    static void setup() throws Exception {
+        setupProject();
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
```

Comment:
should be done as the CwmsDataApiSetupCall.getWebUser()

Reply:
Updated to use CwmsDataApiSetupCall.getWebUser()